cypress/integration/ix-video/react.spec.ts

```diff
@@ -0,0 +1,38 @@
+import {BIG_PLAY_BUTTON_SELECTOR, REACT_PLAYER} from '../../fixtures/selectors';
+
+context('in a React application', () => {
+  beforeEach(() => {
+    cy.visit('/react.html');
+  });
+
+  const host = REACT_PLAYER;
+  const player = `${host} video`;
+  const bigPLayButton = `${host} ${BIG_PLAY_BUTTON_SELECTOR}`;
+
+  it('should render', () => {
+    cy.get(host).should('exist');
+  });
+
+  it('should have a video element', () => {
+    cy.get(player).should('exist');
+  });
+
+  it('should display video controls', () => {
+    cy.get(bigPLayButton).should('exist');
+  });
+
+  it('should display large play button on first render', () => {
+    cy.get(bigPLayButton).should('exist');
+  });
```

Comment:
I suppose if there isn't a straightforward way to differentiate these tests then we should consider removing one?